etl/steps/data/garden/papers/2023-06-07/commodity_prices.py

```diff
@@ -0,0 +1,35 @@
+"""Load snapshot and create a garden dataset."""
+
+import pandas as pd
+from owid.catalog import Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load data from snapshot.
+    #
+    snap = paths.load_snapshot_dependency()
+    df = pd.read_excel(snap.path)
+
+    #
+    # Process data.
+    #
+    # Create a new table and ensure all columns are snake-case.
+    tb = Table(df, short_name=paths.short_name, underscore=True)
```

Comment:
Maybe you could add a country column (with "World" in all rows). Actually you may need it for grapher.

Reply:
Done!